File: Solutions/1773. Count Items Matching a Rule/1773. Count Items Matching a Rule.java
class Solution {
    public int countMatches(List<List<String>> items, String ruleKey, String ruleValue) {
        int res=0,type=0;
        if(ruleKey.equals("color"))
            type=1;
        else if(ruleKey.equals("name"))
            type=2;
        for(var it:items)
            if(it.get(type).equals(ruleValue))
                ++res;
        return res;
    }
}

File: Solutions/1773. Count Items Matching a Rule/1773. Count Items Matching a Rule.py
class Solution:
    def countMatches(self, items: List[List[str]], ruleKey: str, ruleValue: str) -> int:
        res,ty=0,0
        if ruleKey=="color":
            ty=1
        elif ruleKey=="name":
            ty=2
        for it in items:
            if it[ty]==ruleValue:
                res+=1
        return res